sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/EditCasePersonSteps.java

```diff
@@ -80,6 +81,141 @@ public EditCasePersonSteps(final WebDriverHelpers webDriverHelpers) {
               CaseClassification.getUIValueFor(expectedCaseClassification),
               "Case classification value is wrong");
         });
+
+    When(
+        "I set Present condition of Person to ([^\"]*)",
+        (String condition) ->
+            webDriverHelpers.selectFromCombobox(PRESENT_CONDITION_COMBOBOX, condition));
+
+    When(
+        "I check if death data fields are available",
+        () -> {
+          webDriverHelpers.waitUntilElementIsVisibleAndClickable(DATE_OF_DEATH_INPUT);
+          webDriverHelpers.waitUntilElementIsVisibleAndClickable(CASE_OF_DEATH);
+          webDriverHelpers.waitUntilElementIsVisibleAndClickable(DEATH_PLACE_TYPE_COMBOBOX);
+          webDriverHelpers.waitUntilElementIsVisibleAndClickable(DEATH_PLACE_DESCRIPTION);
+        });
+
+    When(
+        "I check if buried data fields are available",
+        () -> {
+          webDriverHelpers.waitUntilElementIsVisibleAndClickable(DATE_OF_BURIAL_INPUT);
+          webDriverHelpers.waitUntilElementIsVisibleAndClickable(BURIAL_CONDUCTOR_INPUT);
+          webDriverHelpers.waitUntilElementIsVisibleAndClickable(BURIAL_PLACE_DESCRIPTION);
+        });
+
+    When(
+        "I fill specific address data",
+        () -> {
+          addressData = caseService.buildAddress();
+          selectCountry(addressData.getCountry());
+          selectRegion(addressData.getRegion());
+          selectDistrict(addressData.getDistrict());
+          selectCommunity(addressData.getCommunity());
+          selectFacilityCategory(addressData.getFacilityCategory());
+          selectFacilityType(addressData.getFacilityType());
+          selectFacility(addressData.getFacility());
+          fillFacilityNameAndDescription(addressData.getFacilityNameAndDescription());
+          fillStreet(addressData.getStreet());
+          fillHouseNumber(addressData.getHouseNumber());
+          fillAdditionalInformation(addressData.getAdditionalInformation());
+          fillPostalCode(addressData.getPostalCode());
+          fillCity(addressData.getCity());
+          selectAreaType(addressData.getAreaType());
+        });
+
+    When(
+        "I click on Geocode button to get GPS coordinates",
+        () -> {
+          webDriverHelpers.clickOnWebElementBySelector(GEOCODE_BUTTON);
+          TimeUnit.SECONDS.sleep(5);
+        });
+
+    When(
+        "I click on save button",
+        () -> {
+          webDriverHelpers.clickOnWebElementBySelector(SAVE_BUTTON);
+          webDriverHelpers.waitForPageLoadingSpinnerToDisappear(40);
```

Comment:
if there's any popup confirming the success action, we can wait for that one as well